content/oci/oci.go

```diff
@@ -526,6 +512,38 @@ func (s *Store) GC(ctx context.Context) error {
 	return nil
 }
 
+// GCIndex reloads the index and updates metadata.
+func (s *Store) GCIndex(ctx context.Context) error {
```

Comment:
This method should be unexported.

```suggestion
func (s *Store) gcIndex(ctx context.Context) error {
```